### PR TITLE
Responsive design -  media queries added

### PR DESCRIPTION
**Pull request by Mukesh2297**

### CODE REVIEW

File: src/app/admin/admin.component.ts [hunk 1]
```diff
@@ -45,9 +45,15 @@ export class AdminComponent implements OnInit {
 
   showSeats = false;
 
+  userData;
```
Comment (gsaileshkumar) [inline src/app/admin/admin.component.ts:48]: remove this

File: src/app/admin/admin.component.ts [hunk 2]
```diff
@@ -45,9 +45,15 @@ export class AdminComponent implements OnInit {
...
+
   constructor(public http: HttpClient, private apiService: ApiService) {}
 
-  ngOnInit(): void {}
+  ngOnInit() {
+    console.log(history.state);
```
Comment (gsaileshkumar) [inline src/app/admin/admin.component.ts:53]: remove console logs everywhere in application

File: src/app/header/header.component.html [hunk 3]
```diff
@@ -6,13 +6,31 @@
...
-  <div>BINGEWATCH</div>
-  <div>
-    <span>Hello {{ userName | titlecase }}</span>
-    <a *ngIf="adminAccess" routerLink="/checkin">Checkin Scanner</a>
-    <a routerLink="/mybookings">My Bookings</a>
-    <a (click)="logout()">Logout</a>
-  </div>
-</mat-toolbar>
+<div class="container">
+    <mat-toolbar color="primary" class="nav-bar">
```
Comment (gsaileshkumar) [inline src/app/header/header.component.html:10]: Component Naming doesn't makes sense.
Header component having Body component inside it.

File: src/app/header/header.component.ts [hunk 4]
```diff
@@ -1,4 +1,6 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
```
Comment (gsaileshkumar) [inline src/app/header/header.component.ts:1]: Where are you using Input?

File: src/app/header/header.component.ts [hunk 5]
```diff
@@ -1,4 +1,6 @@
...
+import { Router, ActivatedRoute } from '@angular/router';
```
Comment (gsaileshkumar) [inline src/app/header/header.component.ts:2]: Remove unused imports everywhere in the app

File: src/app/login/login.component.ts [hunk 6]
```diff
@@ -121,14 +121,15 @@ export class LoginComponent implements OnInit {
 
     this.authService.signIn(params).subscribe((response) => {
       loginResponse = response;
+      console.log(loginResponse.hasError);
```
Comment (gsaileshkumar) [inline src/app/login/login.component.ts:124]: remove console log

File: src/app/seats/seats.component.html [hunk 7]
```diff
@@ -67,25 +67,17 @@
...
-          *ngFor="let showDetails of shows; index as Arrind"
-        >
-          <label>{{ showDetails.hall_name | uppercase }}</label
-          ><br />
-          <button
-            mat-raised-button
-            color="primary"
-            [value]="availability.show_id"
-            *ngFor="
+        <div class="showtime" *ngFor="let showDetails of shows; index as Arrind">
```
Comment (gsaileshkumar) [inline src/app/seats/seats.component.html:70]: What is "Arrind"?
does this word have any meaning?
this is not following any variable name convention.

Follow camelCase everyewhere.

File: src/app/seats/seats.component.ts [hunk 8]
```diff
@@ -54,13 +56,30 @@ export class SeatsComponent implements OnInit {
...
     public http: HttpClient,
     private apiService: ApiService,
-    public matDialog: MatDialog
+    public matDialog: MatDialog,
+    public authService: AuthService,
+    public breakpointObserver: BreakpointObserver
   ) {}
 
   ngOnInit() {
+    console.log('seats component');
```
Comment (gsaileshkumar) [inline src/app/seats/seats.component.ts:70]: remove

File: src/app/seats/seats.component.ts [hunk 9]
```diff
@@ -54,13 +56,30 @@ export class SeatsComponent implements OnInit {
...
+
+    this.breakpointObserver
+    .observe(['(max-width: 700px )'])
+    .subscribe((result) => {
+      if (result.matches) {
+        this.mobileDevice = true;
+      } else {
+        this.mobileDevice = false;
+      }
+      console.log(this.mobileDevice);
```
Comment (gsaileshkumar) [inline src/app/seats/seats.component.ts:80]: remove

File: src/app/services/auth-guard.service.ts [hunk 10]
```diff
@@ -0,0 +1,18 @@
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot, Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AuthGuard implements CanActivate {
```
Comment (gsaileshkumar) [inline src/app/services/auth-guard.service.ts:6]: Delete this file

Comment (Mukesh2297) [reply src/app/services/auth-guard.service.ts:6]: Done

File: src/app/seats/seats.component.ts (hunk 9 above)
Comment (Mukesh2297) [reply src/app/seats/seats.component.ts:80]: Removed console

File: src/app/seats/seats.component.ts (hunk 8 above)
Comment (Mukesh2297) [reply src/app/seats/seats.component.ts:70]: console removed

File: src/app/seats/seats.component.html (hunk 7 above)
Comment (Mukesh2297) [reply src/app/seats/seats.component.html:70]: changed to camelcase-- "hallIndex"

File: src/app/login/login.component.ts (hunk 6 above)
Comment (Mukesh2297) [reply src/app/login/login.component.ts:124]: removed

File: src/app/header/header.component.ts (hunk 5 above)
Comment (Mukesh2297) [reply src/app/header/header.component.ts:2]: unused imports removed everywhere in the app

File: src/app/header/header.component.ts (hunk 4 above)
Comment (Mukesh2297) [reply src/app/header/header.component.ts:1]: removed

File: src/app/admin/admin.component.ts (hunk 2 above)
Comment (Mukesh2297) [reply src/app/admin/admin.component.ts:53]: removed logs everywhere in the app

File: src/app/admin/admin.component.ts (hunk 1 above)
Comment (Mukesh2297) [reply src/app/admin/admin.component.ts:48]: removed

**Pull request merged**